src/caja-spatial-window.c

```diff
@@ -64,6 +64,7 @@
 
 #if !GTK_CHECK_VERSION(3,0,0)
 #define gtk_widget_get_preferred_size(x,y,z) gtk_widget_size_request(x,y)
+#define gtk_vbox_new(X,Y) gtk_box_new(GTK_ORIENTATION_VERTICAL,Y)
```

Comment:
This should be placed in a block several lines lower, along with ```gtk_hbox_new``` define. For now, it breaks GTK+2 build.